test/jdk/jdk/crac/Selector/checkpointLock/Test.java

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019-2021 Azul Systems, Inc.  All Rights Reserved.
```

Comment:
test/jdk/jdk/crac/checkpointLock seems more appropriate (Selector is a set of regression tests for nio.Selector Resource implementation)

Reply:
OK